### ISSUE TITLE
Religions / Belief-systems

### ISSUE THREAD

**Issue by PinPhreek** [widelands/widelands#4285]
**Is your feature request related to a problem? Please describe.**
When I was younger I played a lot of "Settlers IV" where you can build temples to boot strength of your army or just get stuff in return for mana. (Fish, ores, etc.) 

**Describe the solution you'd like**
A possibility for every folk to pray and get mana (or some kind of spiritual energy) and use it in a (non-military) way. (You already can boost the strength of soldiers otherwise). 
For praying you need to have certain resources (Strong Beer, Wine, etc.) 

I know this is a lot I'm asking for and I understand if you say it's too much, it's just an idea...

**Comment by Noordfrees** [comment]
Pretty big change, but a good idea. Could be implemented as an add-on first to see how popular such a change is.

**Comment by Bjoern2Settler** [comment]
I totally agree, its a very nice Idea and maybe gives the game more than the linear building of full heroes.

Very nice Idea +1 maybe they can convert with a low chance units like the evade upgrade
